### DISCUSSION TITLE
Any plans of ELK Detection Engine on SO

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#4510] [Unsupported Versions] [answered]
ELK has their own SIEM called Detection Engine which can be enabled through space>Security. Is there any possibilities that we can see preconfigured ELK SIEM on Security Onion.

**Comment by weslambert** [comment, marked as answer]
There are no plans to provide a pre-configured OOB integration with the SIEM component at this time.  You can enable this,  but there are no guarantees it will work without modification.
